capa-controller-role/README.md

```diff
@@ -0,0 +1,57 @@
+# Cluster API role 
+If you dont know what `INSTALLATION_NAME` value is suppose to be, ask Giant Swarm staff and they will provide it.
+
+# Create role and policy
+
```

Comment:
I'd remove this line and just add a simple explanation on what this is meant to do/create right after the first heading.

```suggestion
```